index.bs

```diff
@@ -485,8 +485,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 : <dfn>Biometric Authenticator</dfn>
 :: Any [=authenticator=] that implements [=biometric recognition=].
 
-: Bound credential
-:: A [=public key credential source=] or [=public key credential=] is said to be <dfn>bound</dfn> to its [=managing
+: <dfn lt="bound">Bound credential</dfn>
+:: A [=public key credential source=] or [=public key credential=] is said to be [=bound=] to its [=managing
```

Comment:
Yep, that's one way to do it -- another is: 
```
: <dfn>Bound credential</dfn>
:: A [=public key credential source=] or [=public key credential=] is said to be [=bound credential|bound=] to its ...
```
...so that if we end up using the term "[=bound credential=]" elsewhere, it just works

Reply:
Hm, ok, I guess that seems fair. Let's go with that.